### DISCUSSION TITLE
Nix-on-droid doesn't install stdenv

### DISCUSSION THREAD

**Discussion by auroraanna** [nix-community/nix-on-droid#182] [Q&A]
Why doesn't the nix-on-droid setup install the nix stdenv? I mean it's literally in the name that should be standard. Why isn't it there?

**Comment by t184256** [comment]
A standard for compiling derivations, not using the OS.

**Comment by auroraanna** [comment]
What? Stdenv is installed on NixOS too by default.

**Comment by auroraanna** [comment]
Not having tools like ping available by default is a bummer.

**Comment by Gerschtli** [comment]
ping is not part of stdenv AFAIK. stdenv is not meant to be installed as a package. What would be the advantage of adding more and more packages to the default installation if adding a package is just adding a line in the config file? Everybody has different use cases and ping may not be useful for other users. So it is hard to find a good set of default packages that match every user really needs.
